### PR TITLE
Migrated text generation screen to fluent UI

### PR DESCRIPTION
**Pull request by ActiveChooN**
Added text generation screen
Screenshot 2024-11-29 at 13 10 40

### CODE REVIEW

**Review by RHeckerIntel — COMMENTED** [review]
Looks damn good! 

There are some state issues when the user stops a response and then asks again causing errors. The error is also completely eaten from the console.

**Comment by RHeckerIntel** [comment]
I think we can just nail out these errors from the main thread instead though. Lets merge when you feel comfortable.

File: lib/pages/text_generation/widgets/assistant_message.dart [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    return Consumer<TextInferenceProvider>(builder: (context, inferenceProvider, child) =>
+      Align(
+        alignment: Alignment.centerLeft,
+        child: Padding(
+          padding: const EdgeInsets.only(bottom: 8),
+          child: Row(
+            crossAxisAlignment: CrossAxisAlignment.end,
+            children: [
+              Padding(
+                padding: const EdgeInsets.only(right: 10, bottom: 36),
```
[the remark this answers is not in the document]
Comment (ActiveChooN) [reply lib/pages/text_generation/widgets/assistant_message.dart]: Yep, good point. Changed

File: lib/pages/text_generation/text_generation.dart [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                        style: ButtonStyle(
+                          shape:WidgetStatePropertyAll(RoundedRectangleBorder(
+                            borderRadius: BorderRadius.circular(4.0),
+                            side:  const BorderSide(color: Color(0XFF545454)),
+                          )),
+                        ),
+                        child: const Text("Close"),
+                        onPressed: () =>  GoRouter.of(context).canPop()
+                          ? GoRouter.of(context).pop()
+                          : GoRouter.of(context).push('/models'),
```
[the remark this answers is not in the document]
Comment (ActiveChooN) [reply lib/pages/text_generation/text_generation.dart]: Fixed

**Pull request merged**